bootstrap/kubeadm/api/v1beta1/kubeadm_types.go

```diff
@@ -260,6 +266,12 @@ type NodeRegistrationOptions struct {
 	// +kubebuilder:validation:Enum=Always;IfNotPresent;Never
 	// +optional
 	ImagePullPolicy string `json:"imagePullPolicy,omitempty"`
+
+	// ImagePullSerial specifies if image pulling performed by kubeadm must be done serially or in parallel.
+	// This option takes effect only on Kubernetes >=1.31.0.
+	// Default: true (defaulted in kubeadm)
```

Comment:
Out of curiosity, why do we want the default to be true for this one?

Reply:
kubeadm defaults to true, I assume because it was the previous behavior

(cc @neolit123)

Reply:
yes, serial is the default in kubeadm and kubelet